docs/install-upgrade/build-wiring.md

```diff
@@ -279,3 +282,321 @@ The dotted line in the diagram shows the traffic flow for remote peering. The tr
 
 A VPC loopback is a physical cable with both ends plugged into the same switch, suggested but not required to be the adjacent ports. This loopback allows two different VPCs to communicate with each other. This is due to a Broadcom limitation.
 
+## Sample Wiring Diagram
+
+The YAML listing below shows a complete wiring diagram. It illustrates how switches
+from a single vendor can be arranged to form a fabric. There are no IP
+addresses or ASN numbers in this listing, the controller creates those as part
```

Comment:
IPs/ASNs currently only created by `hhfab build` and controller can't create them

Reply:
Will change. The controller not being able to do that makes adding a switch after the build a bit interesting as the ASN and IP have to be supplied by hand.

Reply:
yup, it's unfortunate but there is a ton of potentially unexpected behavior if we start to generate it in a runtime :(